src/cards/ha-weather-card.html

```diff
@@ -14,23 +14,29 @@
         }
 
         .attribution {
-          color:grey;
+          color: var(--secondary-text-color);
+          text-align: right;
         }
     </style>
     <google-legacy-loader on-api-load='googleApiLoaded'></google-legacy-loader>
     <ha-card header='[[computeTitle(stateObj)]]'>
       <div class='content'>
-        <template is='dom-repeat' items='[[computeCurrentWeather(stateObj)]]'>
-          <div>[[item.key]]: [[item.value]]</div>
+        <template is='dom-repeat' items='[[computeDisplayAttributes(stateObj)]]' as="attribute">
+          <div>[[formatAttribute(attribute)]]: [[getAttributeValue(stateObj, attribute)]]</div>
         </template>
         <div id='chart_area'></div>
-        <div class='attribution'>[[computeAttribution(stateObj)]]</div>
+        <div class='attribution' hidden$='[[!computeAttribution(stateObj)]]'>[[computeAttribution(stateObj)]]</div>
       </div>
     </ha-card>
   </template>
 </dom-module>
 
 <script>
+(function () {
+  'use strict';
```

Comment:
It is indeed to scope FILTER_KEYS out. `'use strict'` applies some stricter rules to the JavaScript execution run time, it's 👍